04_numeronym/selva/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func nemeronyms(vals ...string) []string {
+	cp := make([]string, len(vals))
```

Comment:
Not a copy, call something else please

Reply:
This logic has been changed now hence this is not applicable.